images/server/Containerfile.centos

```diff
@@ -0,0 +1,54 @@
+FROM quay.io/samba.org/sambacc:latest AS builderscript
+FROM quay.io/centos/centos:stream9 as builder
+COPY --from=builderscript /usr/local/bin/build.sh /usr/local/bin/build.sh
+RUN /usr/local/bin/build.sh --install -v
+
+ARG SAMBACC_VER=master
+ARG SAMBACC_REPO=https://github.com/samba-in-kubernetes/sambacc
+
+# The SAMBACC_VER value controls what version of sambacc we'll be building.
+# This can be a branch name, a tag, or a version hash.  When building tagged
+# versions of samba-containers we should use a sambacc tag.
+# SAMBACC_DISTNAME controls where the script will output a built wheel.
+RUN SAMBACC_DISTNAME=latest \
+    /usr/local/bin/build.sh ${SAMBACC_VER} ${SAMBACC_REPO}
+
+FROM quay.io/centos/centos:stream9
+ARG INSTALL_PACKAGES_FROM=default
+ARG SAMBA_VERSION_SUFFIX=""
+ARG SAMBA_SPECIFICS=daemon_cli_debug_output,ctdb_leader_admin_command
+ARG INSTALL_CUSTOM_REPO=
+
+MAINTAINER John Mulligan <jmulligan@redhat.com>
+
+LABEL org.opencontainers.image.title="Samba container"
+LABEL org.opencontainers.image.description="Samba container"
+LABEL org.opencontainers.image.vendor="Samba in Kubernetes"
+
```

Comment:
Following the same LABELs we use in samba-operrator, please add also org.opencontainers.image.url="https://github.com/samba-in-kubernetes/samba-container

Reply:
@synarete - fwiw, I did this in a separate commit and only changed the centos and fedora based images for a start. opensuse can and probably should be treated separately. what do you think?

Reply:
Fair enough